webpack.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   devServer: {
     contentBase: path.join(__dirname, "dist"),
     compress: true,
-    port: 7000,
+    port: 3000,
```

Comment:
use 7000 as the port number, so that it does not interfere with our backend application